### DISCUSSION TITLE
Ways to make WebUI more alive

### DISCUSSION THREAD

**Discussion by k0stett** [roflcoopter/viseron#536] [Q&A] [answered]
I want to update previews of cameras every second but not ten seconds like it done in code

https://github.com/roflcoopter/viseron/blob/c441207450029fa0a1e6a3e17ef7c822efa9274b/frontend/src/components/camera/CameraCard.tsx#L137

May be there is a config parameter to decrease that time?

**Comment by roflcoopter** [comment, marked as answer]
Good suggestion, will make it configurable instead of hardcoded

**Comment by UndefinedDecoder** [comment]
It's been a while and I still don't see a configuration or setting for this. OP; where is this file located on your system? I run Ubuntu 24.04 LTS container on Proxmox inside a Docker container. I'd be more than happy to alter this value manually but I don't know where the file is. TIA!

**Reply by roflcoopter** [reply]
You can configure it using the `refresh_interval` under `still_image` for the camera.
https://viseron.netlify.app/components-explorer/components/ffmpeg/?_highlight=refresh_interval

```yaml
ffmpeg:
  camera:
    camera_one:
      name: Camera 1
      host: 192.168.XX.X
      port: 554
      path: /Streaming/Channels/101/
      username: !secret camera_one_user
      password: !secret camera_one_pass
      still_image:
        refresh_interval: 1 # refresh every second
